0.6-dev/index.bs

```diff
@@ -0,0 +1,1787 @@
+```
+x = 0.5 * i
+y = 1.2 * j
+```
+
+i.e., the mapping from the first input axis to the first output axis is determined by the first scale parameter.
+
+When rendering transformed images and interpolating, implementations may need the "inverse" transformation - from the output to
+the input coordinate system. Inverse transformations will not be explicitly specified when they can be computed in closed form
+from the forward transformation. Inverse transformations used for image rendering may be specified using the `inverseOf`
+transformation type, for example:
+
+```json
+{
+    "type": "inverseOf",
+    "transformation" : {
+        "type": "displacements",
+        "path": "path/to/displacements",
+    }
+}
+```
+
+Implementations SHOULD be able to compute and apply the inverse of some coordinate transformations when they
+are computable in closed-form (as the [Transformation types](#transformation-types) section below indicates). If an
+operation is requested that requires the inverse of a transformation that can not be inverted in closed-form,
+implementations MAY estimate an inverse, or MAY output a warning that the requested operation is unsupported.
+
+
+#### Matrix transformations
+
+Two transformation types ([affine](#affine) and [rotation](#rotation)) are parametrized by matrices. Matrices are applied to
+column vectors that represent points in the input coordinate system. The first (last) axis in a coordinate system is the top
+(bottom) entry in the column vector. Matrices are stored as two-dimensional arrays, either as json or in a zarr array. When
+stored as a 2D zarr array, the first dimension indexes rows and the second dimension indexes columns (e.g., an array of
+`"shape":[3,4]` has 3 rows and 4 columns). When stored as a 2D json array, the inner array contains rows (e.g. `[[1,2,3],
+[4,5,6]]` has 2 rows and 3 columns).
+
+<div class=example>
+
+For matrix transformations, points in the coordinate system:
+
+```
+{ "name" : "in", "axes" : [{"name" : "z"}, {"name" : "y"}, {"name":"x"}] },
+```
+
+are represented as column vectors:
+
+```
+[z]
+[y]
+[x]
+```
+
+As a result, transforming the point `[z,y,x]=[1,2,3]` with the matrix `[[0,1,0],[-1,0,0],[0,0,-1]]` 
+results in the point [2,-1,3] because it is computed with the matrix-vector multiplication:
+
+```
+[ 0  1  0] [1]   [ 2]
+[-1  0  0] [2] = [-1]
+[ 0  0 -1] [3]   [-3]
+```
+
+</div>
+
+
+### Transformation types
+
+Input and output dimensionality may be determined by the value of the "input" and "output" fields, respectively. If the value
+of "input" is an array, it's length gives the input dimension, otherwise the length of "axes" for the coordinate
+system with the name of the "input" value gives the input dimension.  If the value of "input" is an array, it's
+length gives the input dimension, otherwise it is given by the length of "axes" for the coordinate system with
+the name of the "input".  If the value of "output" is an array, its length gives the output dimension,
```

Comment:
```suggestion
system with the name of the "input" value gives the input dimension.  If the value of "output" is an array, its length gives the output dimension,
```

This sentence is essentially duplicated.

Reply:
thanks: https://github.com/bogovicj/ngff/commit/ede76dbadd099b5e3d29e442521c9a3056e3d281